ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/port-channel-interfaces.j2

```diff
@@ -57,11 +57,12 @@ interface {{ port_channel_interface.name }}
 {%     if port_channel_interface.vlans is arista.avd.defined and port_channel_interface.mode is arista.avd.defined("trunk") %}
    switchport trunk allowed vlan {{ port_channel_interface.vlans }}
 {%     endif %}
-{%     if port_channel_interface.native_vlan is arista.avd.defined and port_channel_interface.mode is arista.avd.defined("trunk") %}
-   switchport trunk native vlan {{ port_channel_interface.native_vlan }}
-{%     endif %}
-{%     if port_channel_interface.mode is arista.avd.defined("trunk phone") and port_channel_interface.native_vlan %}
+{%     if port_channel_interface.mode is arista.avd.defined('trunk') or port_channel_interface.mode is arista.avd.defined('trunk phone') %}
```

Comment:
AVD Style guide suggests to use `in`
```suggestion
{%     if port_channel_interface.mode | arista.avd.default in ['trunk', 'trunk phone'] %}
```

Reply:
just to make sure - `arista.avd.default` ? I was using `defined`

Reply:
https://avd.sh/en/stable/docs/contribution/style-guide.html#syntax-5-test-if-a-variable-in-a-list ok found it